.github/workflows/ci.yml

```diff
@@ -21,11 +21,8 @@ jobs:
       matrix:
         julia-version: ['1.6']
         julia-arch: [x64]
-        os: [ubuntu-latest, windows-latest]
-        include:
-          - julia-version: '1.0'  # test on oldest supported version
```

Comment:
I suggest:

```julia
julia-version: ['1.7']
        julia-arch: [x64]
        os: [ubuntu-latest, windows-latest]
        include:
          - julia-version: '1.6'  # test on oldest supported version
```

Reply:
should it be '1' instead of '1.7' ? it would automatically mean "latest stable release". Also should 1.6 be tested on all OS ?

Reply:
We do it like that in all packages. No strong opinion, but deciding which version is used has its merits.

Usually we only test the lower bound on one system. It is more a warning system that we do not accidentally drop support for some feature. To really do proper testing we would also have to use the lowest package versions.